examples/step-8/doc/results.dox

```diff
@@ -38,7 +38,7 @@ scalars, and Visit then faithfully assumes that this is indeed what it is. In
 other words, once we have written the data as scalars, there is nothing in
 Visit that allows us to paste these two scalar fields back together as a
 vector field. Where we would have to attack this problem is at the root,
-namely in <code>ElasticProblem::output_results</code>. We won't do so here but
+namely in <code>ElasticProblem::output_results()</code>. We won't do so here but
```

Comment:
Should we use `<code>...</code>` here?

Reply:
You mean you're not sure whether we should use `<code>...</code>` here?

I think we do because these are functions in the step-8.cc file which we convert into all doxygen comments. doxygen never sees these functions as actual code it parses, and consequently when you reference these functions like we do here, doxygen (i) doesn't link to them, and (ii) doesn't mark them up as code unless we do so by hand. So I think using the markup as done here is correct.